src/odin/ingestion/qlik/utils.py

```diff
@@ -65,7 +60,9 @@ def re_get_first(string: str, pattern: re.Pattern) -> str:
     return match.group(0)
 
 
-def find_qlik_load_files(table: str, save_local=bool) -> List[Tuple[str, QlikDFM]]:
+def find_qlik_load_files(
+    table: str, source_prefixes: List[str], ds_prefix: str, save_local: bool
+) -> List[Tuple[str, QlikDFM]]:
```

Comment:
Heads up: This is _mostly_ just a pure refactor (if you turn both jobs on), but there is one bugfix which is a behavior change:

Before, this function definition had `save_local=bool`, and the only caller of that function skipped that argument. Since `bool` is truthy, I believe that means that in AWS the `save_local` path was being taken.

I've changed it to `save_local: bool` and passing in `self.save_local` as an argument. This is probably correct, but would result in that `save_local` path no longer being taken in production.